lib/models/ZoneModelWrapped.ts

```diff
@@ -0,0 +1,11 @@
+import ErrorModel from "./ErrorModel.js";
+import ZoneModel from "./ZoneModel.js";
+
+/**
+ * Hetzner DNS zone model
+ * @interface
+ * @extends {ErrorModel}
+ */
+export default interface ZoneModelWrapped extends ErrorModel{
```

Comment:
Pretty much minor issue (Forgot a space in `ErrorModel {`
```suggestion
export default interface ZoneModelWrapped extends ErrorModel {
```